source/mainnet/net/installation/previous-node-downloads.rst

```diff
@@ -0,0 +1,198 @@
+
+`macOS 4.0.11 <https://distribution.concordium.software/macos/signed/concordium-node-4.0.11.pkg>`_
+
+`macOS 3.0.2 <https://distribution.concordium.software/macos/signed/concordium-node-3.0.2.pkg>`_
+
+`macOS 3.0.1 <https://distribution.concordium.software/macos/signed/concordium-node-3.0.1.pkg>`_
+
+`macOS 3.0.0 <https://distribution.concordium.software/macos/signed/concordium-node-3.0.0.pkg>`_
+
+Linux-Docker - Mainnet
+======================
+
+For information about how to run the previous versions of Docker nodes prior to 4.2.3, see :ref:`Run a node with previous Docker node versions<run-previous-docker-version>`.
+
+`4.2.1 <https://distribution.mainnet.concordium.software/tools/linux/concordium-software-linux-4.2.1-0-mainnet.tar.gz>`__
+   - Verification instructions:
+      In a terminal:
+
+      #. Navigate to the download.
+      #. Then paste the following into the terminal: $sha526sum concordium-software-linux-4.2.1-0-mainnet.tar.gz
+      #. Verify that the output matches the SHA256 checksum 8bf746a5efbdc160a562c7c03d9fb7b2f1ab431877aad066187d4bd8354294e5.
+
+`4.1.1 <https://distribution.mainnet.concordium.software/tools/linux/concordium-software-linux-4.1.1-1-mainnet.tar.gz>`__
+   - Verification instructions:
+      In a terminal:
+
+      #. Navigate to the download.
+      #. Then paste the following into the terminal: $sha526sum concordium-software-linux-4.1.1-1-mainnet.tar.gz
+      #. Verify that the output matches the SHA256 checksum eb5c7e0573b0778589ae035373a253330e367ad281916739ed9f5104fd08c22b.
+
+`3.0.2 <https://distribution.mainnet.concordium.software/tools/linux/concordium-software-linux-3.0.2-0-mainnet.tar.gz>`__
+   - Verification instructions:
+      In a terminal:
+
+      #. Navigate to the download.
+      #. Then paste the following into the terminal: $sha526sum concordium-software-linux-3.0.2-0-mainnet.tar.gz
+      #. Verify that the output matches the SHA256 checksum 733600b800f7a184152453a5aa52f6d0d50101a698804a957eb82c47ff2396f8.
+
+`3.0.1 <https://distribution.mainnet.concordium.software/tools/linux/concordium-software-linux-3.0.1-0-mainnet.tar.gz>`__
+   - Verification instructions:
+      In a terminal:
+
+      #. Navigate to the download.
+      #. Then paste the following into the terminal: $sha526sum concordium-software-linux-3.0.1-0-mainnet.tar.gz
+      #. Verify that the output matches the SHA256 checksum 9939f3f002a899645d3698488e5d99c9683a475f4d33197f8cab70a9cb39f2b3.
+
+`3.0.0 <https://distribution.mainnet.concordium.software/tools/linux/concordium-software-linux-3.0.0-0-mainnet.tar.gz>`_
+   - Verification instructions:
+      In a terminal:
+
+      #. Navigate to the download.
+      #. Then paste the following into the terminal: $sha526sum concordium-software-linux-3.0.0-0-mainnet.tar.gz
+      #. Verify that the output matches the SHA256 checksum 897ef39807eb449945b0498def00ca14d2762be8f9d206a62f1d9cd97f2beec0.
```

Comment:
This won't work since the tool downloads the "active" docker image, which is always the latest. So these dowloads are not useful for running old versions of the node.

Moreover we are deprecating this distribution in favor of the new docker one, so I would argue we should not have download links for this anymore.

Reply:
@smh1001 Do you agree?